nautobot/docs/development/release-checklist.md

```diff
@@ -64,7 +84,14 @@ Ensure that continuous integration testing on the `develop` branch is completing
 
 ### Update Version and Changelog
 
-Update the `VERSION` constant in `settings.py` to the new release version and annotate the current data in the release notes for the new version. Commit these changes to the `develop` branch.
+Update the package version in `pyproject.toml` to the new release version. For example, if we wanted to update the package to `2.0.0`:
+
+```no-highlight
+$ poetry version 2.0.0
+Bumping version from 1.0.0 to 2.0.0
+```
```

Comment:
Do we want to instead recommend using `poetry version [patch|minor|major]` to automatically select the new version number appropriately?

Reply:
Yes. I actually didn't know that was a thing.